### PR TITLE
Add common aliases for all write-* commands default message parameter

### PR DESCRIPTION
**Pull request by KevinMarquette**
## PR Summary
add common aliases (`message`, `msg`) to Write-* commands default message parameter. This allows the user to rename `Write-Host` to `Write-Debug` or `Write-Verbose` to `Write-Information` without having to rename parameters (if they are specified). 

* add `Message` alias to the `MessageData` parameter for `Write-Information`
* add  `Msg` and `Message` alias to the `Object` parameter for `Write-Host`
* add tests that use Message and Msg aliases on Write-Information and Write-Host

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [X] Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] User facing Documentation PR added: https://github.com/PowerShell/PowerShell-Docs/pull/2141
    - [X] Issue filed - Issue link: https://github.com/PowerShell/PowerShell-Docs/issues/2134
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by KevinMarquette** [comment]
Changed title to WIP because the build failed.

**Comment by iSazonov** [comment]
I think that aliases should be added _only_ to solve some specific scenarios - what is the scenarios? In all other cases, this may cause problems now or in the future when it is necessary to remove these aliases and this will be an breaking change.

**Comment by KevinMarquette** [comment]
I was trying to resolve some inconsistencies between similar commands. 

`Write-Information`, `Write-Verbose`, `Write-Debug` and `Write-Error` should have the same default parameter names so they are easily interchangeable. `Write-Host` and `Write-Output` could easily be argued to be part of this set.

One could make the argument that I took that too far with this change and I could dial it back to that specific scenario. I could exclude `Write-Output` and `Write-EventLog` to minimize the scope of this change.

**Comment by SteveL-MSFT** [comment]
@PowerShell/powershell-committee reviewed this and agree it makes sense for consistency for Message/Msg for Write-Host, Write-Verbose, Write-Warning, Write-Error, Write-Debug, Write-Information, but should not use `InputObject` as an alias nor change Write-Output nor Write-EventLog.

**Comment by adityapatwardhan** [comment]
@KevinMarquette Thank you for your contribution. Please update the PR according to the PowerShell-Committee recommendations.

**Comment by iSazonov** [comment]
@KevinMarquette Please update PR summary too.

**Comment by KevinMarquette** [comment]
@adityapatwardhan  Thank you, PR info is updated

**Comment by adityapatwardhan** [comment]
@JamesWTruher Do you think we need a test for this?

**Comment by adityapatwardhan** [comment]
@JamesWTruher Ping..

**Comment by JamesWTruher** [comment]
we probably should have a test for this, the expectation is that if it is present, we should be checking to be sure it's not removed

**Review by dantraMSFT — CHANGES_REQUESTED** [review]
Write-Information.md and Write-Host.md need to be updated to reflect the new aliases.  Either create an issue or consider submitting a PR with the changes.
They can be found under reference\7\Microsoft.PowerShell.Utility in PowerShell/PowerShell-Docs repo.

**Comment by KevinMarquette** [comment]
@dantraMSFT I did not see a reference\7 folder in that repo. Do I create it and add the docs there? In the meantime, I created an issue for the docs.

@JamesWTruher I added tests that execute those aliases and other parameters that were untested.

@adityapatwardhan I removed the WIP status from this pull request.

**Comment by iSazonov** [comment]
@KevinMarquette It is another repo - PowerShell/PowerShell-Docs.

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1 [hunk 1]
```diff
@@ -96,5 +104,15 @@ Describe "Stream writer tests" -Tags "CI" {
...
        }
+
+       It "Write-Information works with <Name>" -TestCases:$testInfoData {
+            param($Command, $returnCount, $returnValue)
+            $ps.AddScript($Command).Invoke()
+
+            $result = $ps.Streams.Information
+
+            $result.Count | Should Be $returnCount
+            (Compare-Object $result $returnValue -SyncWindow 0).length -eq 0 | Should Be $true
```
Comment (adityapatwardhan) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1]: This should be as follows for a better error message when it fails:

`(Compare-Object $result $returnValue -SyncWindow 0).length | Should Be 0`

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by dantraMSFT** [comment]
@KevinMarquette: Sorry for the typo; it should be a 6, not 7.  
reference\6\Microsoft.PowerShell.Utility

File: test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1 (hunk 1 above)
Comment (KevinMarquette) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1]: I like this suggestion. I made the same change in the Write-Host Tests too.

**Comment by KevinMarquette** [comment]
@dantraMSFT documentation added in https://github.com/PowerShell/PowerShell-Docs/pull/2141

@adityapatwardhan the corrections to the tests have been made.

**Review by adityapatwardhan — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@dantraMSFT Please update your review.

**Review by dantraMSFT — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@KevinMarquette Thank you for your contribution!

**Pull request merged**